package.json

```diff
@@ -11,9 +11,11 @@
   },
   "dependencies": {
     "@tanstack/react-query": "^5.64.1",
+    "@types/react-router-dom": "^5.3.3",
+    "lucide-react": "^0.474.0",
```

Comment:
please remove this package since we do not need it. we can implement icons using svgs